build/AddressBook/dll/AddressBook.vcxproj

```diff
@@ -30,6 +30,9 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)\dllmain.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <ClangCompile Include="..\..\..\Frameworks\AddressBook\NSDate+AddressBookAdditions.mm" />
```

Comment:
"Since it is a category, it will require either a category force-include or manual inclusion in the .dll and .UnitTests.exe" he mentioned that in another comment

---
In reply to: [75209330](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75209330) [](ancestors = 75209330)